ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -507,41 +536,30 @@ public void replicaThreadTest() throws InterruptedException, IOException {
       Map<PartitionId, List<ByteBuffer>> bufferListNode1 = new HashMap<PartitionId, List<ByteBuffer>>();
       Map<PartitionId, List<ByteBuffer>> bufferListNode2 = new HashMap<PartitionId, List<ByteBuffer>>();
       for (int i = 0; i < partitionIds.size(); i++) {
-        List<MessageInfo> messageInfoListLocalReplica = new ArrayList<MessageInfo>();
-        List<ByteBuffer> messageBufferListLocalReplica = new ArrayList<ByteBuffer>();
-
-        List<MessageInfo> messageInfoListRemoteReplica2 = new ArrayList<MessageInfo>();
-        List<ByteBuffer> messageBufferListLocalReplica2 = new ArrayList<ByteBuffer>();
-
-        for (int j = 0; j < 10; j++) {
-          BlobId id = new BlobId(partitionIds.get(i));
-          ByteBuffer byteBuffer = constructTestBlobInMessageFormat(id, 1000, random);
-          long streamSize = byteBuffer.limit();
-          messageInfoListLocalReplica.add(new MessageInfo(id, streamSize));
-          messageInfoListRemoteReplica2.add(new MessageInfo(id, streamSize));
-          messageBufferListLocalReplica.add(byteBuffer);
-          messageBufferListLocalReplica2.add(byteBuffer);
-        }
+        ArrayList<MessageInfo> localInfoList = new ArrayList<>();
+        ArrayList<ByteBuffer> localBufferList = new ArrayList<>();
 
-        // add additional messages to replica 2
-        for (int j = 10; j < 15; j++) {
-          BlobId id = new BlobId(partitionIds.get(i));
-          ByteBuffer byteBuffer = constructTestBlobInMessageFormat(id, 1000, random);
-          long streamSize = byteBuffer.limit();
-          messageInfoListRemoteReplica2.add(new MessageInfo(id, streamSize));
-          messageBufferListLocalReplica2.add(byteBuffer);
-        }
+        ArrayList<MessageInfo> remoteInfoList = new ArrayList<>();
+        ArrayList<ByteBuffer> remoteBufferList = new ArrayList<>();
+
+        Arrays.asList(localInfoList, remoteInfoList);
```

Comment:
This is being done inline now (in the next statement), so this line should have been removed.